webpack/ForemanInventoryUpload/ForemanInventoryConstants.js

```diff
@@ -0,0 +1,4 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const INVENTORY_PAGE_TITLE = __('Red Hat Inventory Uploads');
+export const INVENTORY_BASE_URL = 'INVENTORY_BASE_URL';
```

Comment:
I think this will render all URLs to start with `INVENTORY_BASE_URL/`. I don't think it's desired.

Reply:
ahah, good catch.. forgot to change it :rofl: